### PR TITLE
CORE-652: GEN Transfer Reader/Writer for FileService

### PR DESCRIPTION
**Pull request by EBGToo**

### CODE REVIEW

File: generic/BRGeneric.c [hunk 1]
```diff
@@ -204,6 +209,23 @@ genTransferSerialize (BRGenericTransfer transfer, size_t *bytesCount) {
     return transfer->handlers.getSerialization (transfer->ref, bytesCount);
 }
 
+static size_t
+genTransferSetHash (const void *transferPtr) {
+    BRGenericTransfer transfer = (BRGenericTransfer) transferPtr;
+    return genTransferGetHash (transfer).value.u32[0];
```
Comment (carlcherry) [inline generic/BRGeneric.c]: Isn't this suppose to be "SetHash"

Comment (EBGToo) [reply generic/BRGeneric.c]: It is actually correct - but about the worst possible naming...

Comment (EBGToo) [reply generic/BRGeneric.c]: Addressed issue, renamed functions.

**Comment by EBGToo** [comment]
@carlcherry This is ready for review (moved from 'draft')

**Pull request merged**